### PR TITLE
Add "getting started" documentation

### PR DESCRIPTION
**Pull request by bqqbarbhg**
Wrote some documentation about the setup!

Read the formatted doc at: https://github.com/ktk-2005/Feedbacker-Forum/blob/getting-started/docs/getting-started.md

### CODE REVIEW

**Comment by bqqbarbhg** [comment]
Added everyone in case you have some opinions about the general setup now

**Comment by bqqbarbhg** [comment]
Also note that this documentation assumes that `npm run watch` support (#7) will be merged.

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+| --- | --- |
+| `lint` | Verify code style, use `npm run lint -- --fix` to fix mistakes automatically |
+
+Rarely manually needed NPM scripts for server and client:
+
+| Command | Description |
+| --- | --- |
+| `lint:ci` | Lint like the CI does, checking the line ending format. This will likely fail on Windows as Git likes to check out `\r\n` line endings. |
+
+## Client
```
Comment (oamaok) [inline docs/getting-started.md:36]: As there is already second level header `Client and server`, should this and the `Server` below be third level headers?

**Review by shahruzmannan — APPROVED** [review]

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| Command | Description |
+| --- | --- |
+| `test:api` | Run automated API tests, requires having the server running at `localhost:8080`. |
+
+## Docker
+
+The *docker/* directory contains subdirectories for both development and production Docker setups.
+They can be built using `docker-compose`.
+
+### Deveolpment
```
Comment (HenrikHelenius) [inline docs/getting-started.md:81]: typo

File: docs/server-config.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+# Server configuration
+
+The API server loads configuration from a JSON file. Some values may be overridden by
+environment variables. You can specify the configuration file to use with the `-c`/`--config` flags.
+
+## Confiuration JSON format
```
Comment (HenrikHelenius) [inline docs/server-config.md:7]: typo

File: docs/server-config.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `port` | `Integer` | Port to bind the API server to, overridden by env `APP_SERVER_PORT`. |
+| `dev` | `Boolean` | If set to true the server runs in development mode serving also static content. |
```
Comment (HenrikHelenius) [inline docs/server-config.md:12]: good doc

File: docs/getting-started.md (hunk 1 above)
Comment (bqqbarbhg) [reply docs/getting-started.md:36]: I don't really know what the cleanest way to format this is, there should be a section with general things shared between the client and the server, and ones dedicated to each. I kinda feel like they belong to the same heading level but might be wrong.

@HenrikHelenius opinions? You kept the Google Docs pretty clean

Comment (HenrikHelenius) [reply docs/getting-started.md:36]: Should be sub level headers to Client and server 

I.e. :
> Client and server > Client + Server

**Pull request merged**